application/gqlgen/schema.graphql

```diff
@@ -846,31 +846,65 @@ input UpdateUserPreferencesInput {
 }
 
 """
-A Watch for a given location. Requests with a destination near the watch location will trigger a notification to
-the watch creator. Other types of Watches (e.g. keyword search) may be created in future versions of WeCarry.
+A Watch for a given location. New requests matching all of the given criterium will generate a new
```

Comment:
"... all of the given criteria ..."  (i.e. plural, right)?

Reply:
Yep.